src/Tribe/Repositories/Event.php

```diff
@@ -1465,26 +1463,48 @@ public function handle_order_by( $order_by ) {
 
 		$timestamp_key = 'TIMESTAMP(mt1.meta_value)';
 
-		$by_event_start_date     = isset( $check_orderby['event_date'] )
-		                           || in_array( 'event_date', $check_orderby, true );
-		$by_event_start_date_utc = isset( $check_orderby['event_date_utc'] )
-		                           || in_array( 'event_date_utc', $check_orderby, true );
-		$by_organizer            = isset( $check_orderby['organizer'] )
-		                           || in_array( 'organizer', $check_orderby, true );
-		$by_venue                = isset( $check_orderby['venue'] ) || in_array( 'venue', $check_orderby, true );
-		$by_timestamp_key        = isset( $check_orderby[ $timestamp_key ] )
-		                           || in_array( $timestamp_key, $check_orderby, true );
-
-		if ( $by_event_start_date || $by_event_start_date_utc ) {
-			$this->order_by_date( $by_event_start_date_utc );
-		} elseif ( $by_organizer ) {
-			$this->order_by_organizer();
-		} elseif ( $by_venue ) {
-			$this->order_by_venue();
-		} elseif ( $by_timestamp_key ) {
-			$this->filter_query->orderby( $timestamp_key );
-		} else {
-			$this->query_args['orderby'] = $order_by;
+		$after    = false;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces

<sub>hash: 86f61c31abdb196da4f71c2c7c155000</sub>